controllers/provisioning_controller.go

```diff
@@ -222,7 +219,7 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		return ctrl.Result{}, errors.Wrap(err, "failed to create Inspector password")
 	}
 
-	// If Metal3 Deployment already exists, do nothing.
+	// If Metal3 Deployment already exists and managed by MAO, do nothing.
```

Comment:
I think we need a function named ExistingDeploymentNotOwedByUS() - also get rid of the "Get"

Reply:
It's going to disappear soon anyway, but I agree this isn't very well named.

Reply:
The logic around this is getting refined as part of https://github.com/openshift/cluster-baremetal-operator/pull/65. And, please feel free to propose better names.